printer_mods/hartk1213/Voron0_MGN9C_X_Axis/Readme.md

```diff
@@ -0,0 +1,28 @@
+MGN9C X axis
+============
+
+This replaces the MGN7H rail on the V0.0 X axis with a single MGN9C

```

Comment:
This says V0.0, but the CAD images show V0.1 - is it compatible with both?  Or just V0.1?

Reply:
Ah good catch lol..it has been updated.. thanks Kilo